core/tests/protractor_desktop/topicAndStoryViewer.js

```diff
@@ -155,6 +159,14 @@ describe('Story viewer functionality', function() {
     await topicAndStoryViewerPage.expectUncompletedLessonCountToBe(1);
   });
 
+  it('should work properly for story viewer page.', async function() {
```

Comment:
Can you make this test name more descriptive?

Reply:
Done.